io.github.tomluchowski.OpenDungeonsPlus.yml

```diff
@@ -0,0 +1,144 @@
+
+  # SFML module (Simple and Fast Multimedia Library)
+  - name: sfml
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+    sources:
+      - type: archive
+        url: https://github.com/SFML/SFML/archive/refs/tags/2.5.1.tar.gz
+        sha256: 438c91a917cc8aa19e82c6f59f8714da353c488584a007d401efac8368e1c785
+
+
+  # OGRE module, which will include ImGui sources
+  - name: ogre
+    buildsystem: cmake-ninja
+    config-opts:
+            - -DCMAKE_BUILD_TYPE=Release
+            - -DOGRE_CONFIG_THREAD_PROVIDER=std
+            - -DOGRE_CONFIG_THREADS=3
+            - -DOGRE_STATIC=FALSE
+            - -DOGRE_BUILD_DEPENDENCIES=FALSE
+            - -DOGRE_BUILD_RENDERSYSTEM_GL=TRUE
+            - -DOGRE_BUILD_RENDERSYSTEM_GL3PLUS=TRUE
+            - -DOGRE_RESOURCEMANAGER_STRICT=0
+            # stuff we dont need
+            - -DOGRE_BUILD_RENDERSYSTEM_GLES2=FALSE
+            - -DOGRE_BUILD_TOOLS=FALSE
+            - -DOGRE_BUILD_SAMPLES=FALSE
+            - -DOGRE_BUILD_PLUGIN_FREEIMAGE=FALSE
+            - -DOGRE_BUILD_PLUGIN_EXRCODEC=FALSE
+            - -DOGRE_BUILD_PLUGIN_BSP=FALSE
+            - -DOGRE_BUILD_PLUGIN_PCZ=FALSE
+            - -DOGRE_BUILD_PLUGIN_DOT_SCENE=FALSE
+            - -DOGRE_BUILD_COMPONENT_JAVA=FALSE
+            - -DOGRE_BUILD_COMPONENT_VOLUME=FALSE
+            - -DOGRE_BUILD_COMPONENT_PAGING=FALSE
+            - -DOGRE_BUILD_COMPONENT_TERRAIN=FALSE
+            - -DOGRE_BUILD_COMPONENT_PROPERTY=FALSE
+            - -DOGRE_BUILD_COMPONENT_MESHLODGENERATOR=FALSE
+            - -DOGRE_BUILD_COMPONENT_HLMS=FALSE
+            - -DOGRE_ENABLE_PRECOMPILED_HEADERS=OFF
+
+    sources:
+      - type: archive
+        url: https://github.com/OGRECave/ogre/archive/refs/tags/v13.6.5.tar.gz
+        sha256: 857f74d18c4bb484f0e31c288b8d27837615a59c3da2b3d8304496c83ef669cd  # Replace with actual checksum
+
+      # Download ImGui as an additional source to be used in the OGRE build
+      - type: archive
+        url: https://github.com/ocornut/imgui/archive/refs/tags/v1.87.tar.gz
+        sha256: b54ceb35bda38766e36b87c25edf7a1cd8fd2cb8c485b245aedca6fb85645a20  # Replace with actual checksum
+        dest: imgui-1.87  # This will extract ImGui to /run/build/imgui
+
+
+  # CEGUI module
+  - name: cegui
+    buildsystem: cmake-ninja
+    config-opts:
+            - -DCMAKE_BUILD_TYPE=Release
+            - -DCMAKE_CXX_FLAGS=-std=c++11
+            - -DCEGUI_BUILD_RENDERER_OPENGL=FALSE
+            - -DCEGUI_BUILD_RENDERER_OPENGLES=FALSE
+            - -DCEGUI_BUILD_RENDERER_OPENGL3=FALSE
+            - -DCEGUI_BUILD_IMAGECODEC_FREEIMAGE=FALSE # we use ogre
+            - -DCEGUI_OPTION_DEFAULT_IMAGECODEC=OgreRenderer-0
+            - -DCEGUI_SAMPLES_ENABLED=FALSE
+            - -DCEGUI_BUILD_PYTHON_MODULES=OFF
+            - -DCEGUI_LIB_INSTALL_DIR=lib
+    sources:
+      - type: git
+        url: https://github.com/tomluchowski/cegui.git
+        tag: v0
```

Comment:
this is a branch, not a tag.

Reply:
hmm what format do you expect ?
Why tags are so important to you ? so no one dare to change it after sometime ? 

 I suppose it will be easier to download an archive ....

Reply:
a commit or tag is expect. because we expect the build to reproduce. which a branch doesn't allow.

Reply:
`tag` mean tag. but you put a branch. Ideally it's a `tag` and a `commit`. or just a `commit`.